docs/source/command_line.rst

```diff
@@ -366,6 +366,10 @@ Here are some more useful flags:
   also currently ignores functions with an empty body or a body that is
   just ellipsis (``...``), since these can be valid as abstract methods.
 
+- ``--strict-boolean`` will make any non-boolean conditions errors.
```

Comment:
Could you expand this documentation a little bit more?  I think someone reading it for the first time (without being exposed to the idea before) would have difficulty understanding it as is.  (I don't think it needs a lot more -- maybe just one or two sentences.)

I'd also like to suggest two slight tweaks. First, "conditions" isn't a super common term (in my experience), so I'd consider writing something like "non-boolean expressions in conditionals" instead.  Second, instead of using `5` as a constant, I'd consider saying something more general, like "`if x` is disallowed if `x` has any type other than `bool`."